File: programmers/level2/위장.py
def solution(clothes):
    answer = 1
    
    #구조화
    clothesCnt = {}
    for c in clothes:
        cur = c[1]
        if cur in clothesCnt:
            clothesCnt[cur] += 1
        else:
            clothesCnt[cur] = 1
    
    #조합 계산
    for val in clothesCnt.values():
        answer *= val + 1

    return answer - 1

File: programmers/level2/위장.cpp
#include <string>
#include <vector>
#include <unordered_map>
using namespace std;

int solution(vector<vector<string>> clothes) {
    unordered_map<string, int> m;
    for(auto c: clothes){
    	m[c[1]] += 1;
    }
    int ans = 1;
    for(auto it=m.begin(); it!=m.end(); it++){
    	ans *= (it->second + 1);
    }
    return ans - 1;
}
